internal/api/backresthandler.go

```diff
@@ -858,6 +859,29 @@ func (s *BackrestHandler) GetSummaryDashboard(ctx context.Context, req *connect.
 		DataPath:   env.DataDir(),
 	}
 
+	if hostname, err := os.Hostname(); err != nil {
+		zap.S().Warnf("failed to determine hostname: %v", err)
+	} else if hostname != "" {
+		response.Hostname = hostname
+	}
+
+	if version := systemOSVersion(); version != "" {
+		response.OsVersion = version
+	}
+
+	if s.orchestrator != nil {
+		response.ResticPath = s.orchestrator.ResticBinary()
+		if response.ResticPath != "" {
```

Comment:
This one is quite slow as well-- let's defer it to the future PR as well, with that change I think this LGTM to merge!